README.md

```diff
@@ -111,7 +111,7 @@ If you want to deploy the operator on OpenShift you will have to provide the env
 
 See [here](deploy/crds/mongodb.com_v1_mongodb_openshift_cr.yaml) for an example of how to provide the required configuration for a MongoDB ReplicaSet.
 
-See [here](deploy/operator_openshift.yaml) for an example of how to configure the Operator deployment.
+See [here](deploy/openshift/operator_openshift.yaml) for an example of how to configure the Operator deployment.
```

Comment:
moved the openshift example into another directory so it is still possible to do. `kubectl apply -f deploy/` without deploying the operator twice